docs/quantization.md

```diff
@@ -82,17 +82,17 @@ Here are some examples of quantization configurations
   ```
 * Only quantize linear layers
   ```
-  --quantize '{"linear:a8w4dq": {"groupsize" : 256}}'
+  --quantize '{"linear:a8w4dq": {"groupsize" : 32}}'
```

Comment:
Just to jog my memory: Is the only reason we use 32 here instead of 256 is so that the stories model works?